hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMDBCheckpointServlet.java

```diff
@@ -320,16 +355,44 @@ private boolean getFilesForArchive(DBCheckpoint checkpoint,
         hardLinkFiles, sstFilesToExclude,
         new HashSet<>(), excluded, copySize,
         compactionLogDir.getOriginalDir().toPath());
+  }
+
+  private void estimateTotals(Path dir,
+     AtomicInteger totalFiles,
+     AtomicLong totalSize) throws IOException {
+    List<Path> subDirs = new ArrayList<>();
+    Stream<Path> paths = Files.list(dir);
+    totalFiles.addAndGet(paths
+        .filter(p -> {
+          File f = p.toFile();
+          if (f.isDirectory()) {
+            subDirs.add(p);
+            return false;
+          }
+          return f.getName().toLowerCase().endsWith(".sst");
+        })
+        .map(f -> {
+          try {
+            totalSize.addAndGet(Files.size(f));
+          } catch (IOException e) {
+            throw new RuntimeException(e);
+          }
+          return f.toString();
+        }).collect(Collectors.toList()).size());
```

Comment:
Collecting into a list just to get its size is not needed, `Stream` has `count()`.

(but `estimateTotals` can be completely removed, so not applicable now)